packages/embark-ui/package.json

```diff
@@ -102,7 +102,7 @@
     "react-json-view": "1.19.1",
     "react-redux": "5.1.1",
     "react-router-dom": "4.3.1",
-    "react-scripts": "1.1.5",
+    "react-scripts": "2.1.5",
```

Comment:
I am not even sure we use this package as we exported.
And I think really upgrading to 2 would actually cause error and we would need to re-export